portfolio.js

```diff
@@ -20,3 +20,156 @@ menuLinks.forEach((link) => {
+    keywords: ['html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://www.example.com/',
+    sourceLink: 'https://github.com/example/',
+    keyClass: 'keys',
+  },
+  {
+    id: '6',
+    class: 'card six',
+    name: 'Profesional Art Printing Data',
+    img: 'images/screenshoot2.png',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry standard.',
+    descriptionPopUp: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    keywords: ['html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://www.example.com/',
+    sourceLink: 'https://github.com/example/',
+    keyClass: 'keys',
+  },
+];
+
+const projectsSection = document.getElementById('works');
+
+projectsData.forEach((project) => {
+  const projectElement = document.createElement('div');
+  projectElement.classList.add('project');
+
+  projectElement.innerHTML = `
+    <div class="card-container">
+      <div class="${project.class}">
+          <h3>${project.name}</h3>
+          <p>${project.description}</p>
+          <ul class="${project.keyClass}">${project.keywords.map((keyword) => `<li>${keyword}</li>`).join('')}
+          </ul>
+          <button class="${project.buttonClass} project-details" onclick="showModal(${project.id})">See Project</button>
+      </div>
+    </div>
+  `;
+
+  projectsSection.appendChild(projectElement);
+});
+
+const closeModal = () => {
+  const myModal = document.getElementById('myModal');
+  myModal.style.display = 'none';
+};
+
+const showModal = (id) => {
+  const myModal = document.getElementById('myModal');
+  myModal.style.display = 'flex';
+
+  const project = projectsData[id];
+
+  myModal.innerHTML = `
+    <div>
+      <a class="closeBtn">&times;</a> 
+      <h1>${project.name}</h1>
+      <p>${project.descriptionPopUp}</p>
+      <img src="${project.img}" />
+      <ul>${project.keywords.map((keyword) => `<li>${keyword}</li>`).join('')}
+      </ul>
+      <div class="btns">
+        <a class="liveBtn" target="_blank" href='${project.liveLink}'>See live</a>
+        <a class="sourceBtn" target="_blank" href='${project.sourceLink}'>See source</a>
+      </div>
+    </div>
```

Comment:
I have this on my screen right now... what should I do?

![image](https://user-images.githubusercontent.com/43345162/224173422-862611a3-7d75-4d48-967b-bb6463dd1009.png)

Reply:
@vickymarz Im asking because I dont know how to fix something that I cannot see... do you have any recommendation?

Reply:
@MarcoDDM, I think you should submit again and add a screenshot based on the previous review. THe issue might be from my PC. The next reviewer will access it from his PC and give you feedback Dont too worry 😄

Reply:
@MarcoDDM Also make sure that you actually pushed the commit that has the latest changes. You can just adjust something from your end and push again. THis will help you ascertain that the commits have been pushed. KIndly do this 👍🏽

Reply:
Let me show you a little bit of this:

https://www.loom.com/share/257d1f27470e4dc394da0e8fdb80873b

Reply:
@MarcoDDM, good job. As I have earlier mentioned. try to make a small change from any part of your code and push it again to GitHub so as to be sure that the latest changes have been pushed. Also, add the loom video as a comment while submitting again. 

Also this might also be an issue from my PC so don't worry. YOu will still get the best review possible 👍🏽